langchain4j-infinispan/src/test/java/dev/langchain4j/store/embedding/infinispan/InfinispanEmbeddingStoreIT.java

```diff
@@ -7,16 +7,18 @@
 import dev.langchain4j.store.embedding.EmbeddingStoreIT;
 import org.infinispan.client.hotrod.configuration.ClientIntelligence;
 import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+import org.infinispan.commons.util.Version;
 import org.infinispan.server.test.core.InfinispanContainer;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
 
 import static org.infinispan.server.test.core.InfinispanContainer.DEFAULT_PASSWORD;
 import static org.infinispan.server.test.core.InfinispanContainer.DEFAULT_USERNAME;
+import static org.infinispan.server.test.core.InfinispanContainer.IMAGE_BASENAME;
 
 class InfinispanEmbeddingStoreIT extends EmbeddingStoreIT {
 
-    static InfinispanContainer infinispan = new InfinispanContainer();
+    static InfinispanContainer infinispan = new InfinispanContainer(IMAGE_BASENAME + ":" + Version.getVersion());
```

Comment:
@langchain4j the default image used will be 15.0 which is always the latest, but docker does not pull it if you have it locally

Reply:
adding this maps to 15.0.0.Final instead of 15.0

Reply:
@karesti thank you, it works now!